templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -22,6 +23,24 @@ configuration file (`.zuliprc`) which you will need for integrations that you wo
 to use with this bot. You can click on the download
 (<i class="icon-vector-download-alt"></i>) icon to download your `.zuliprc` file.
 
+### Bot types
+
+You can create three types of bots:
+
+![Bot types](/static/images/help/bot_types.png)
+
+* **Generic bot:** A generic bot can send *and* receive messages.
+
+* **Incoming webhook:**  The API keys for such bots are limited to
+  only sending messages and can't receive messages. Thus, this bot
+  type lessens the security risks associated with exposing such API
+  keys to third-party services, since they can't be used to read
+  messages.
+
+* **Outgoing webhook:** Bots of this type are the same as a
+  **Generic bot**, except **Outgoing webhooks** have an extra field
+  for the base URL of the outgoing webhook service to `POST` to.
```

Comment:
Maybe I'm just bad, but I had to read this sentence several times to figure out what it meant. (It kind of reads like the base URL is doing the POSTing)